logger/types.go

```diff
@@ -1,7 +1,7 @@
 package logger
 
 const (
-	JsonLogFormat = iota
+	JSONLogFormat = iota
```

Comment:
This is a breaking change. Not sure we should change that. @kumarabd ?
It can be exclude from the linter by using a comment like `//nolint:<check>`.

Reply:
I really want to change this, even if this means I replicate this across places and update.😅 But you're right, better to comment out the lint check.